### PR TITLE
[to #268] fix: tf_blade still links to shared cuDNN.

### PR DESCRIPTION
**Pull request by qiuxiafei**
Now torch_blade and tf_blade has same linking behavior on static TensorRT and cuDNN.

### CODE REVIEW

**Review by Orion34-lanbo — APPROVED** [review]
LGTM

File: docker/dev/Dockerfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ RUN bash /install/scripts/find-fastest-apt.sh && \
 
 ARG DEVICE=cu110
 RUN bash /install/scripts/install-tensorrt.sh "${DEVICE}"
-RUn bash /install/scripts/patch-cu102.sh "${DEVICE}"
+RUn bash /install/scripts/patch-cuda.sh "${DEVICE}"
```
Comment (Orion34-lanbo) [inline docker/dev/Dockerfile:19]: Is `RUn` right?

File: docker/runtime/Dockerfile.pytorch [hunk 2]
```diff
@@ -6,7 +6,7 @@ ADD ./docker/scripts /install/scripts
 RUN bash /install/scripts/find-fastest-apt.sh && \
         apt-get install -y -qq wget curl unzip
 RUN bash /install/scripts/install-tensorrt.sh "${DEVICE}"
-RUn bash /install/scripts/patch-cu102.sh "${DEVICE}"
+RUn bash /install/scripts/patch-cuda.sh "${DEVICE}"
```
Comment (Orion34-lanbo) [inline docker/runtime/Dockerfile.pytorch:10]: ditto

File: docker/dev/Dockerfile (hunk 1 above)
Comment (qiuxiafei) [reply docker/dev/Dockerfile]: It works but shouldn't be so. fixed.

File: docker/runtime/Dockerfile.pytorch (hunk 2 above)
Comment (qiuxiafei) [reply docker/runtime/Dockerfile.pytorch:10]: fixed

**Pull request merged**